dds/DCPS/RTPS/RtpsDiscovery.cpp

```diff
@@ -595,6 +621,30 @@ RtpsDiscovery::Config::discovery_config(ACE_Configuration_Heap& cf)
                               ACE_TEXT("[rtps_discovery/%C] section.\n"),
                               string_value.c_str(), rtps_name.c_str()), -1);
           }
+        } else if (name == "SedpHeartbeatPeriodMinimum") {
```

Comment:
Does it make sense to assert / complain if minimum > maximum? Or maybe if backoff factor < safety factor?

Reply:
The backoff factor vs. safety factor doesn't really matter, they can be anything (should be greater than 1).  I will update the code so they can reverse the limits and it will still work properly.